src/test/java/com/amihaiemil/docker/RtContainerTestCase.java

```diff
@@ -25,12 +25,17 @@
  */
```

Comment:
@amihaiemil you're leaving a puzzle for finishing implementation of the rest of the `Container` API, but I feel we need another one specifically for adding more test cases for `start` and `inspect`